changes/gem5-resources.tex

```diff
@@ -0,0 +1,13 @@
+\subsection[gem5 resources]{gem5 resources\footnote{by Mahyar Samani}}
+
+gem5 resources are the set of inputs used to conduct computer systems architecture
+research on known-good configurations using gem5. They include anything from the config
+files used to build a linux kernel that works with gem5 to the config scripts that describe
+the computer system to be simulated. The provided resources have been tested with gem5-20
+and their working status and initial statistics have been documented. They could be used to
```

Comment:
refer the working status page: https://www.gem5.org/documentation/benchmark_status/ (fyi, the page is not there yet)